src/bp/ui-shared/src/Form/FormFields/Upload.tsx

```diff
@@ -104,7 +86,7 @@ const Upload: FC<UploadFieldProps> = props => {
             inputProps={{
               id: 'node-image',
               name: 'nodeImage',
-              accept: 'image/*',
+              accept: filter || `${type || 'image'}/*`,
```

Comment:
nice 👌  ! where does the filter come from ? jsonSchema filters ?

Reply:
Yes!